src/Servers/Kestrel/test/InMemory.FunctionalTests/TlsListenerTests.cs

```diff
@@ -21,7 +21,7 @@
 
 namespace InMemory.FunctionalTests;
 
-public class TlsListenerMiddlewareTests : TestApplicationErrorLoggerLoggedTest
+public class TlsListenerTests : TestApplicationErrorLoggerLoggedTest
```

Comment:
We can probably add a cancellation test here as well. Set the handshake timeout to something small like 1 millisecond and check that the request was canceled.

Reply:
added